src/packages/searchbar/searchbar.taro.tsx

```
@@ -55,137 +63,214 @@ export const SearchBar: FunctionComponent<
+      setTimeout(() => {
+        setInnerTag(tag)
+      }, 100)
+    },
+    [onBlur, tag, value]
+  )
+
+  const clearaVal = useCallback(
+    (event: ITouchEvent) => {
+      if (disabled || readOnly) return
+      setValue('')
+      onChange && onChange('')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)